docs/migrating/migrating_from_ez_publish.md

```diff
@@ -42,7 +42,16 @@ If you are coming directly from legacy (4.x), you need to follow the best practi
 - Rewrite custom admin modules to use the new Platform/Symfony stack
     - And if you do this while on 5.x, you can use several of the [available legacy migration features](https://doc.ez.no/display/EZP/Legacy+code+and+features) to make the new code appear in legacy admin
 
-See Upgrade documentation on how to perform the actual upgrade: [Upgrade (eZ Publish Platform page)](https://doc.ez.no/display/EZP/Upgrade)
+See Upgrade documentation on how to perform the actual upgrade: [Upgrade (eZ Publish Platform page)](https://doc.ez.no/display/EZP/Upgrade).
+
+!!! caution "Avoid exception when migrating the database"
+
+    If you plan to migrate from from eZ Publish through eZ Publish Platform 5.4 to eZ Platform and further, an exception may occur when you try to migrate the database while it contains internal drafts of Landing Pages.
+    This can happen because such drafts do not have an expected row in the `ezcontentobject_name` table.<a id="migration_exception"></a> 
```

Comment:
what does this `<a id="migration_exception"></a>` do? Anchors should usually have some content inside them so they're visible/clickable.

Reply:
No need to be either visible or clickable - it is merely a trick to help redirect from the other occurrence of this warning to the detailed recommendations.